code/__DEFINES/admin.dm

```diff
@@ -38,12 +38,6 @@
 
 #define R_DEFAULT R_AUTOLOGIN
 
-#if DM_VERSION > 512
-#error Remove the flag below , its been long enough
-#endif
-//legacy , remove post 512, it was replaced by R_POLL
-#define R_REJUVINATE	2

```

Comment:
If you're removing this, also get rid of:
https://github.com/tgstation/tgstation/blob/23a45889ed8618be2c512977618d717daea0a553/code/modules/admin/admin_ranks.dm#L88-L90

Reply:
removed